### ISSUE TITLE
Indicate when the tier of a machine is too low to run a recipe in machines.

### ISSUE THREAD

**Issue by Zorbatron** [GregTechCEu/GregTech#2500]
### GregTech CEu Version

v2.8.8

### Related Problem

I see a good amount of people in the Nomi CEu discord asking why their extruder isn't working and 95% of the time it's because they're using an LV extruder when almost all of the recipes are MV.

### Your Solution

Have the Gregtech icon in singleblock machines (maybe multis too?) flash yellow when a recipe is unable to run when the voltage requirement isn't met. A tooltip for when you hover over the icon saying something along the lines of: "Machine tier too low for recipe!" would be good too.

**Comment by PSR2144-3933** [comment]
agree. this can solve massive amounts of new gtceu players didn't catch the voltage tier requirement
